acceptance-tests/src/test/java/tech/pegasys/signers/secp256k1/tests/multikey/AzureBasedTomlLoadingAcceptanceTest.java

```diff
@@ -50,9 +51,7 @@ void azureSignersAreCreatedAndExpectedAddressIsReported(@TempDir Path tomlDirect
 
     setup(tomlDirectory);
 
-    assertThat(
-            signerProvider.availablePublicKeys().stream()
-                .map(pk -> Bytes.wrap(pk.getValue()).toUnprefixedHexString()))
-        .containsOnly(PUBLIC_KEY_HEX_STRING);
+    assertThat(signerProvider.availablePublicKeys().stream().map(PublicKey::toEthHexString))
```

Comment:
_should_ theEthHexString be _on_ the public key, or a helperfunction? I _prefer_ static helper-function, as it keeps "PublicKey" really clean and single purpose.